ServerCore/Pages/Teams/Index.cshtml

```diff
@@ -11,77 +11,90 @@
 <h2>All Teams</h2>
 
 <p>
-    @if (Model.Event.IsTeamRegistrationActive && Model.EventRole == ModelBases.EventRole.admin)
+    @if (Model.EventRole == ModelBases.EventRole.admin)
     {
+        if (!Model.Event.IsTeamRegistrationActive)
+        {
+            <div class="alert alert-danger" role="alert">
+                This event is not currently open for registration.
+            </div>
+        }
+        else if (Model.Event.MaxNumberOfTeams <= Model.Teams.Count)
```

Comment:
Super small nit: I think this gets easier to read if you reverse it like you had for players above:
(Model.Teams.Count >= Model.Event.MaxNumberOfTeams)
It reads closer to the way you'd say it in a sentence:
"If the number of teams is greater than or equal to the max allowed number of teams"
The sentence the other way sounds awkward because the maximum is a constant so it's odd to check it relative to something else:
"If the max allowed number of teams is less than or equal to the number of teams"
Anecdotally at least, it feels like it took me longer to understand / double check this one than the ones above

Reply:
On the plus side I absolutely love the super descriptive error messages - thanks for updating all of those :D

Reply:
Fixed in both places, thanks!